schema/dereferenced-release-schema.json

```
@@ -6606,7 +6606,7 @@
           },
           "value": {
             "title": "Value",
-            "description": "The value of this award. There may be more than one award per procurement. A negative value indicates that the contract(s) resulting from this award may involve payments from the supplier to the buyer (commonly used in concession contracts). This field should not contain the value of a framework agreement; those should be given in estimatedValue or maximumValue instead.",
+            "description": "The value of this award. Typically, this is the value of the bid being awarded. There may be more than one award per procurement. A negative value indicates that the contract(s) resulting from this award may involve payments from the supplier to the buyer (commonly used in concession contracts). This field should not contain the value of a framework agreement; those should be given in estimatedValue or maximumValue instead.",
```

```suggestion
            "description": "The value of the award. Typically, this is the value of the bid being awarded. There may be more than one award per procurement. A negative value indicates that the contract(s) resulting from this award may involve payments from the supplier to the buyer (commonly used in concession contracts). This field should not contain the value of a framework agreement; those should be given in estimatedValue or maximumValue instead.",
```
Discussed in https://docs.google.com/spreadsheets/d/1WaN5cdU5UxoCMkq8Od5zlLj68-ckn1lnzWpt_DfzqHk/edit#gid=0 how most fields use the pattern "The <object> of the <object>" rather than "The <object> of this <object>"

---

Done in https://github.com/open-contracting/standard/pull/1656/commits/06ae3361228489d5c3f1b9bfff3fad09823029fd